### PR TITLE
Expand "add religion to place of worship" quest

### LINKED ISSUE #1708: Adding religion quest to more features
The religion quest could easily be exended to include at least the following additional features:
1. [cemetery](https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dcemetery)
2. [grave yard](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dgrave_yard)
3. [monastery](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dmonastery)

I implemented a f[irst draft for cemeteries](https://github.com/Dosenpfand/StreetComplete/commit/596326b6b66530f2509783739ef2e447343b4927) and can expand it and open a pull request if desired.

### General
Affected tag(s) to be modified/added: [religion](https://wiki.openstreetmap.org/wiki/Key:religion)
Question asked: **What religion is represented at this cemetery?**

### Checklist
Checklist for quest suggestions (see guidelines):
- [X] 🚧 To be added tag is established and has a useful purpose
- [X] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [X] 🐿️ Easily answerable by everyone from the outside but a survey is necessary 
- [X] 💤 Not an overwhelming percentage of elements have the same answer (No spam)
- [X] 🕓 Applies to a reasonable number of elements (Worth the effort)

### Ideas for implementation

**Element selection:** `"nodes, ways, relations with landuse = cemetery and !religion"`

**Metadata needed:** N/A

**Proposed GUI:** 
Similar to adding religion to places of worship and wayside shrines.

### PR DESCRIPTION
**Pull request by Dosenpfand**
fixes #1708 
- Include monasteries
- Include features without name

### CODE REVIEW

File: app/src/main/java/de/westnordost/streetcomplete/quests/religion/AddReligionToPlaceOfWorship.kt [hunk 1]
```diff
@@ -7,12 +7,25 @@ import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataAndGeometr
...
 class AddReligionToPlaceOfWorship(o: OverpassMapDataAndGeometryDao) : SimpleOverpassQuestType<String>(o) {
 
-    override val tagFilters =
-        "nodes, ways, relations with amenity = place_of_worship and !religion and name"
+    override val tagFilters = """
+        nodes, ways, relations with 
+        (
+            (amenity = place_of_worship)
+            or
+            (amenity = monastery)
```
Comment (westnordost) [inline app/src/main/java/de/westnordost/streetcomplete/quests/religion/AddReligionToPlaceOfWorship.kt]: You do not need the brackets here

**Comment by westnordost** [comment]
Nice, well done!
Small nitpicking there, but won't block the merge.

**Comment by Dosenpfand** [comment]
Thanks!
I incorporated your finding.

**Comment by westnordost** [comment]
Nice, thank you!

**Pull request merged**